shared/js/background/events/3p-tracking-cookie-blocking.js

```diff
@@ -61,10 +61,10 @@ function dropTracking3pCookiesFromRequest (request) {
     if (tab && request.type !== 'main_frame') {
         const requestIsTracker = trackerutils.isTracker(request.url)
         if (requestIsTracker && !tab.site.isFeatureEnabled('trackingCookies3p')) {
-            return { responseHeaders }
+            return { requestHeaders }
```

Comment:
Was this broken code? `responseHeaders` is not defined in this scope?

Reply:
I don't see this change in the diff. Could be something I typed wrong and fixed in a commit?